packages/shared/src/lib/featureManagement.ts

```diff
@@ -116,17 +116,6 @@ export class Features<T extends FeatureValue = string> {
     'Activate companion',
   );
 
-  static readonly PostCardVersion = new Features('post_card_version', 'v1', [
```

Comment:
Just checking these where nowhere used in tracking that you might have removed right?

Reply:
I have not removed any trackEvent afair